### ISSUE TITLE
Possible Glitch

### ISSUE THREAD

**Issue by Kenarik** [jaquadro/StorageDrawers#1110]
I have had a few issues that seem minor in the last few weeks and I have narrowed it down to possibly this mod. Let me explain, I am on a Chromebook, running through Linux.  I moved to Forge because I saw my kid using this mod and I loved it.  

Sometime in the last few weeks, No, I can not narrow it down any further case it's so subtle I didn't even notice until it got really bad. There are times where I either can not put the items down to use them, or I can not even take them out of the drawer they are in. If I log off and log back on, I can then do it, It's also affecting putting boats into the water, at least for me.  That's the one I noticed first.

Nothing ever disappears from the game, I just can't use it. When I log back in, it's sitting there in the drawer like it hadn't ever been taken out.  I thought you might want to at least check the mod. I know you haven't updated it in a bit but it only started happening in the last few weeks.

If, I really hope, it is not your mod and if it is Minecraft itself then I will have to let them know.

**Comment by Kenarik** [comment]
Unfortunately others have tried on a server I am on and this was the issue. While I will still play with it on my singleplayer world until this gets replicated and fixed we simply can not have it on the server. Which is a huge let down for all of us.

I'm running 1.20.1 and Forge and yes I have the latest version for that. I may step down a version and see if that helps.
If it does I will comment again.

**Comment by NekoCharly** [comment]
Good morning!, were you able to fix it? It's the same on a server I'm on, there are 4 of us but the same glitch happens to 2 people

**Comment by Kenarik** [comment]
No, stepping down to 1.20.1-12.0.1 did not help. I am about to try on my world 12.0.0 and see if that helps.
